### ISSUE TITLE
Update Brazil Time Zone(s)

### ISSUE THREAD

**Issue by d3fz** [ericpaulbishop/gargoyle#727]
Please update/add the [time zone(s) currently being used in Brazil](https://www.timeanddate.com/time/zone/brazil). 

I'm on UTC-3 (BRT). Note my router current date & time.

![8f8cdcce-06a3-4340-8e91-c76ae6d1cd2e](https://user-images.githubusercontent.com/22578839/36113091-ce19eb66-1012-11e8-9563-e0f881b87b06.png)
![22fe3d3e-3f25-4773-a167-ec08b443d538](https://user-images.githubusercontent.com/22578839/36113097-d3b40a84-1012-11e8-995e-9dbb1597681a.png)

Not exactly sure where I should place those values, so I'm leaving this one to the experts.

Thank you in advance. 👍

**Comment by lantis1008** [comment]
Sure.
Is there no such thing as Brazil East and Brazil West time now?
It is all just Brazil Time (with Daylight Savings)?

**Comment by d3fz** [comment]
Pretty much, yes. There are actually four time zones, stretching from UTC-2 to UTC-5. 

UTC-3: this is largest time zone, comprising eastern and central areas (most important). 

Brazil West would be UTC-4.

**Comment by lantis1008** [comment]
This should cover it. Changes the data behind Brazil East.
https://github.com/lantis1008/gargoyle/commit/0f29375c613dcca1e7488ed12bc13153219b81d9

Note: `DST starts 1st Sunday in November. DST ends 3rd Sunday in Feb. End date is correct in most years except when this sunday is Carnival. Then there is nothing we can do`

**Comment by d3fz** [comment]
That's great, thank you. 👍 

>Note: DST starts 1st Sunday in November. DST ends 3rd Sunday in Feb. End date is correct in most years except when this sunday is Carnival. Then there is nothing we can do

Understood.

**Comment by d3fz** [comment]
Hi @lantis1008 ,

Based on your commit, I've made some additional changes to "Brazil West" (aka Amazon Time) timezone string.

```
- "UTC-04:00 TiZ.BRW"		"BRWST4BRWDT,M10.3.0/0,M2.5.0/0"	"south-america"
+ "UTC-04:00 TiZ.AMT"		"AMT4AMST,M11.1.0/0,M2.3.0/0"	"south-america"
``` 
Could you please add this to your changes so it goes altogether?

TIA 👍

**Comment by lantis1008** [comment]
It would be handled similar to the Australia case. 
We have New South Wales that is defined as AEST10AEDT. We also have Queensland which is just AEST all year. 

So let’s do:
AMT all year
AMT/AMST
BRT all year
BRT/BRST

To define these they’ll be called “Brazil, x”. Can you please come up with a good identifier for each of those? Either a state or major city that observes those times and someone would look at it and say “that’s the Timezone I’m in because I have the same as that place”. If you need to list more than 1 that’s ok, but preferably only for the ones with daylight savings as they will have their own entry. 

Hope this makes sense

**Comment by lantis1008** [comment]
That is, we are removing “Brazil East/West”, unless those are the most appropriate names :+1:

**Comment by d3fz** [comment]
>So let’s do:
AMT all year
AMT/AMST
BRT all year
BRT/BRST

That would cover it all.

>To define these they’ll be called “Brazil, x”. Can you please come up with a good identifier for each of those? Either a state or major city that observes those times and someone would look at it and say “that’s the Timezone I’m in because I have the same as that place”.

AMT all year - Brazil, Manaus
AMT/AMST - Brazil, Cuiaba
BRT all year - Brazil, Fortaleza
BRT/BRST - Brazil, Brasilia

>That is, we are removing “Brazil East/West”

Agreed. 

Just out of curiosity, in which TZ you're at?

**Comment by lantis1008** [comment]
Thanks, I’ll do those this evening. 

I’m in New South Wales, Australia which is UTC+10 (AEST), but we are currently in Daylight Savings (AEDT). 
It is currently 9:30am Thursday 15th.

**Comment by lantis1008** [comment]
https://github.com/lantis1008/gargoyle/commit/10595ef15b4ff2dcccca0dd0b5bd5a28b038e06a

That should close out this issue now?

**Comment by d3fz** [comment]
Absolutely.

Thank you for your time. 👍

**Comment by d3fz** [comment]
The issue was addressed in #732.

Closed.
